app/controllers/items_controller.rb

```diff
@@ -1,37 +1,37 @@
 class ItemsController < ApplicationController
-    before_action :authenticate_user!
-    before_action :set_group
+  before_action :authenticate_user!
+  before_action :set_group
 
-    def index
-        @items = Group.find(@group.id).items.order(created_at: :desc)
-    end
+  def index
+    @items = Group.find(@group.id).items.order(created_at: :desc)
+  end
 
-    def new
-        @group = Group.find(params[:group_id])
-        @item = @group.items.build
-    end
+  def new
+    @group = Group.find(params[:group_id])
+    @item = @group.items.build
+  end
 
```

Comment:
- [ ] Kindly design the **_form page_** for adding a new **_transaction_** to look identical to that of the design layout provided for forms. In the design layout, a form page for registering a user was provided. Kindly use the design layout for other form pages in your application. Designing a page based on a wireframe or sample layout ensures consistent and accurate implementation of the intended user experience, reducing ambiguity and misinterpretation. 🖱️ 🖥️ 


| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/3b12348c-bc07-4b98-8773-0148298cbe5a) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/1167219c-4a3b-48e4-b19d-22ebb21c5edd) |